### PR TITLE
Animate new explore feed content in from the top

### PR DESCRIPTION
**Pull request by joewalsh**
https://phabricator.wikimedia.org/T162167

### CODE REVIEW

**Comment by joewalsh** [comment]
retest this please

**Comment by joewalsh** [comment]
retest this please

File: Wikipedia/Code/MWKDataStore.m [hunk 1]
```diff
@@ -475,15 +475,15 @@ - (NSManagedObjectContext *)feedImportContext {
         _feedImportContext = [[NSManagedObjectContext alloc] initWithConcurrencyType:NSPrivateQueueConcurrencyType];
         _feedImportContext.parentContext = _viewContext;
         _feedImportContext.mergePolicy = NSMergeByPropertyStoreTrumpMergePolicy;
-        [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(backgroundContextDidSave:) name:NSManagedObjectContextDidSaveNotification object:_feedImportContext];
+        //[[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(backgroundContextDidSave:) name:NSManagedObjectContextDidSaveNotification object:_feedImportContext];
```
Comment (amrox) [inline Wikipedia/Code/MWKDataStore.m]: Can this and the following commented out line be deleted?

Comment (joewalsh) [inline Wikipedia/Code/MWKDataStore.m]: Accidental commit - I was shortcutting the save so that I could test. Thanks for catching this

**Pull request merged**